test/packages/deploy_terraform/_dev/deploy/docker/docker-compose.yml

```diff
@@ -0,0 +1,3 @@
+version: '2.3'
+services:
+  hello_world:
```

Comment:
Nit. Is this file needed here? I guess it doesn't do any harm though.

Reply:
leftover from copying over the `deploy_docker` module, thank you for catching this.